articles/user-guide/libraries/standard/prelude.md

```diff
@@ -291,34 +278,34 @@ for (index in 0..Length(qs)-1)
 return rs;
 ```
 
-## Extension Functions and Operations ##
+## Extension functions and operations
```

Comment:
The concept of extension functions and operations is no longer really used; that exposed an implementation detail about the .NET-based runtime for Q# that has now been abstracted away much better, such that users aren't required to memorize what parts of the Q# library are implemented in pure Q# and which parts are delegated to the runtime.

Reply:
@cgranade - Thanks, this is great feedback. I figured these topics were old enough that significant updates to the content would be needed.  This will help with the next phase.